README.md

```diff
@@ -35,6 +35,15 @@ For instance, to validate against version `1.0` of the JSON schema:
 => true
 ```
 
+You can also use the `LaaCrimeSchemas::Validator` class to validate a document against a the MAAT application JSON schema.
```

Comment:
Small typo `against a the` -  a can be removed

Reply:
good spot!